### PR TITLE
feat: created history video

### PR DESCRIPTION
**Pull request by Shubhamkr2610**
image
image

### CODE REVIEW

File: src/components/videocard/VideoCard.js [hunk 1]
```diff
@@ -11,7 +11,8 @@ export const VideoCard = ({
   categoryName,
 }) => {
   return (
-    <>
+    <>  
+    {/* <div > */}
```
Comment (Rockyraven) [inline src/components/videocard/VideoCard.js]: Remove unwanted code

**Review by arjun-computer-geek — APPROVED** [review]
Code are good and tradable

**Review by sumitk-singh — APPROVED** [review]
Codes are clean and easy to read.
Good to merge.

**Review by Rockyraven — APPROVED** [review]
All goods for me
You can go ahead

File: src/components/videocard/VideoCard.js (hunk 1 above)
Comment (Shubhamkr2610) [reply src/components/videocard/VideoCard.js]: removed

**Pull request merged**